lib/elasticity/version.rb

```diff
@@ -1,3 +1,3 @@
 module Elasticity
-  VERSION = '2.9.6'
+  VERSION = '2.9.7'
```

Comment:
Is this a big enough change to go to 2.10.0? 3.0.0?

Reply:
Sure.